### DISCUSSION TITLE
How does Ansible determine a given collection's modules?

### DISCUSSION THREAD

**Discussion by SHxKM** [ansible-collections/overview#190] [Q&A] [answered]
If we look at the `sensu` namespace as an example, Ansible Galaxy can "auto-detect" the content of a given collection:
<img width="1163" alt="Screen Shot 2021-12-08 at 10 10 56" src="https://user-images.githubusercontent.com/6573474/145172275-f7752171-39d4-4ca2-9fad-3039566f6b1c.png">

Where is Galaxy getting this information from? I saw in some collections there are some [metadata tags](https://github.com/ansible-collections/amazon.aws/blob/main/README.md?plain=1#L30) in the README, but it is absent from others, so I'm not sure how should I structure our docs/README (if it's pulled from there).

**Comment by felixfontein** [comment, marked as answer]
It looks at which files are in `plugins/modules/` in the tarball that has been uploaded.
